src/AppInitializer.tsx

```diff
@@ -200,22 +200,19 @@ class AppInitializer extends React.Component<Props, AppInitializerState> {
         }
       }
     }
-    let color = 'white'
-    let background = '#4db7ad'
+    let color = '#4db7ad'
+    let background = 'white'
     let name = ''
     let icon = ''
     const footerElement = document.querySelector('#footer')
     if (footerElement) {
       color = window
-        .getComputedStyle(footerElement, null)
-        .getPropertyValue('color')
-      background = window
         .getComputedStyle(footerElement, null)
         .getPropertyValue('background-color')
     }
```

Comment:
You delete the background statement and not the color state in `if (footerElement)` block. Is this intentional?

Reply:
Yep, it now uses the footer background color as the themed color in the back-bar.  And the back-bar background color is just white now.